Common/WALinuxAgent-2.0.16/waagent

```diff
@@ -2903,6 +2915,13 @@ class Logger(object):
         """
         self.LogWithPrefix("ERROR:", message)
 
+    def _get_log_prefix(self, prefix):
+        """
+        Generates the log prefix with timestamp+'prefix'.
+        """
+        t = time.localtime()
```

Comment:
Using UTC time instead of local time ensures consistency of timestamps. I suggest using str(datetime.datetime.utcnow())

Reply:
I don't mind making the change but I didn't want to change existing logic (this is just refactoring the duplicated code into its own method)!

Reply:
I think keeping the local time is okay. As the logs would reside on the box, it is okay. Even WAAgent logs in local time on the box.